internal/plugin.bzl

```diff
@@ -53,3 +70,9 @@ def postcss_plugin(
         srcs = srcs + data + deps,
         visibility = visibility,
     )
+
+    postcss_plugin_info(
+        name = "%s.info" % name,
+        node_require = node_require,
+        visibility = visibility,
+    )
```

Comment:
Could we make `postcss_plugin` itself a rule that has the `node_require` info as well as the `srcs`, `data`, and `deps` that are encapsulated in the filegroup above?

Reply:
Yep that's definitely possible. Check out this prototype: https://github.com/bazelbuild/rules_postcss/commit/ef55044ef84d1b5695479fb141337e0233933dbb#diff-57ca3dad7c7acaf3d1151aeb57a83e5a

But because of b/147932115 I think it's cleaner to have this PR be an intermediate step.

Reply:
Oops, I realize that prototype still has the macro postcss_plugin.

But I expect that we could replace postcss_plugin with just the plugin info rule (i.e. get rid of the macro) in that prototype, then change `internal/binary.bzl:66` back to `deps = deps` instead of `deps = deps + plugin_infos`.

I'll give that a shot.

Reply:
Done. Here's the [prototype](https://github.com/bazelbuild/rules_postcss/compare/nodejs-info-in-plugin#diff-57ca3dad7c7acaf3d1151aeb57a83e5a).

This was pretty quickly hacked together and the optional next step would be to remove the wrapper macro, but it does work.

Reply:
I like that prototype a lot. I think it's worth removing the wrapper macro and using that as the official implementation of `postcss_plugin`.

Reply:
That's totally the aim. Let's merge this first for b/147932115 and then merge that separately to resolve #19?

Reply:
SGTM